File: src/com/leek/wars/client/entities/responses/AisResponse.java
package com.leek.wars.client.entities.responses;

import java.util.Iterator;
import java.util.List;

import com.leek.wars.client.entities.AI;

public class AisResponse extends Response implements Iterable<AI> {

	private List<AI> ais;

	
	public List<AI> getAis() {
		return ais;
	}
	public void setAis(List<AI> ais) {
		this.ais = ais;
	}

	
	@Override
	public Iterator<AI> iterator() {
		return ais.iterator();
	}
	
}

File: src/main/kotlin/com/leek/wars/bot/entities/responses/AisResponse.kt
package com.leek.wars.bot.entities.responses

import com.leek.wars.bot.entities.AI
import com.leek.wars.bot.entities.Folder

class AisResponse : Response(), Iterable<AI?> {
    private var ais: MutableList<AI>? = null
    var folders: List<Folder>? = null
    fun getAis(): List<AI>? {
        return ais
    }

    fun setAis(ais: MutableList<AI>?) {
        this.ais = ais
    }

    override fun iterator(): MutableIterator<AI> {
        return ais!!.iterator()
    }
}